src/App.js

```diff
@@ -1,16 +1,15 @@
 import React from "react";
-import {
-  BrowserRouter as Router,
-  Switch,
-  Route,
-} from "react-router-dom";
-import HomePage from './screens/HomePage';
-import LoginPage from './screens/LoginPage';
-import AboutUsPage from './screens/AboutUsPage';
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
```

Comment:
Let's keep the max line length to 120

Reply:
Fixed with [a76f4af](https://github.com/Suvink/let-me-react/pull/15/commits/a76f4af6e4662e78f6b4c5ee4e0a47a630c237dd)